Package.resolved

```diff
@@ -238,10 +238,10 @@
     {
       "identity" : "xcodegraph",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/tuist/XcodeGraph.git",
+      "location" : "https://github.com/darrarski/XcodeGraph.git",
       "state" : {
-        "revision" : "0e44220403553ce7f20a79e49873b492302b4650",
-        "version" : "0.5.0"
+        "branch" : "add-class-prefix-to-project",
+        "revision" : "627830f87824cf7b1bc62298d86475d32a0ec1e2"
```

Comment:
You can revert these changes and point your PR to the version 0.7.0 or XcodeGraph :)

Reply:
I've updated the dependency so it points to `tuist/XcodeGraph` version `.upToNextMajor(from: "0.7.0")`. Unfortunately, there is a lot of changes between `0.5.0` and `0.7.0`, causing many build errors in the base Tuist repo. Not sure if I'm able to resolve them all (I tried, but it touches a lot of files, which I'm not familiar with).

Could we hold on, until the bump to `XcodeGraph` `0.7.0` will be made, possibly in a separate PR, by someone more experienced in Tuist's codebase?